### PR TITLE
Multiplayer layout 6

### PR DESCRIPTION
**Pull request by kapapp**
Linked up multiplayer page with 2 empty sub-activities (friend, random)

### CODE REVIEW

**Review by JorisMonnet — APPROVED** [review]
You could import espresso to avoid calling it every time but really minor, nice interface

File: app/src/androidTest/java/ch/epfl/sdp/blindwar/ui/multi/MultiPlayerRandomActivityTest.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    @After
+    fun tearDown() {
+        Intents.release()
+    }
+
+    @Test
+    fun onCreate() {
+        //Placeholder
```
Comment (Marty99-dev) [inline app/src/androidTest/java/ch/epfl/sdp/blindwar/ui/multi/MultiPlayerRandomActivityTest.kt:24]: Should we use a " " (space) after the "//" ?

File: app/src/main/java/ch/epfl/sdp/blindwar/ui/MainMenuActivity.kt [hunk 2]
```diff
@@ -27,6 +28,11 @@ class MainMenuActivity : AppCompatActivity() {
         startActivity(intent)
     }
 
+    fun multiButton(view: View) {
+        val intent = Intent(this, MultiPlayerActivity::class.java)
```
Comment (Marty99-dev) [inline app/src/main/java/ch/epfl/sdp/blindwar/ui/MainMenuActivity.kt:32]: Maybe do it inline

File: app/src/main/java/ch/epfl/sdp/blindwar/ui/multi/MultiPlayerActivity.kt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        val intent = Intent(this, MultiPlayerFriendActivity::class.java)
+        startActivity(intent)
+    }
+
+    /**
+     * Button logic towards the multiplayer with random person activity
+     *
+     * @param view
+     */
+    fun randomButton(view: View) {
```
Comment (Marty99-dev) [inline app/src/main/java/ch/epfl/sdp/blindwar/ui/multi/MultiPlayerActivity.kt:37]: Same comments for both functions above

**Review by Marty99-dev — APPROVED** [review]

**Pull request merged**